Code/Engine/ECS/Systems/PlayerControllerSystem.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include <Engine/ECS/Registry.h>
+#include <Engine/ECS/Components/TransformComponent.h>
+#include <Engine/InputManager.h>
+
+namespace sad
+{
+	class PlayerControllerSystem
+	{
+	public:
+		PlayerControllerSystem();
+		~PlayerControllerSystem();
```

Comment:
Do we need to have constructors and destructors defined if all methods on the system are `static`?